src/main/scala/cromwell/engine/workflow/CallMetadataBuilder.scala

```diff
@@ -37,8 +37,8 @@ object CallMetadataBuilder {
     def extract(job: Any): BackendValues = {
       job match {
         case ji: LocalJob => BackendValues("Local")
-        case ji: JesJob => BackendValues("JES", jobId = Option(ji.jesId.toString), status = ji.jesStatus)
```

Comment:
This was and still is an `Option`.  Before I was `toString`ing an `Option` and wrapping that in an `Option`, which works about as poorly as you'd imagine.  :smile: